src/test/obj_reorder_basic/obj_reorder_basic.c

```diff
@@ -105,7 +105,7 @@ main(int argc, char *argv[])
 	UT_ASSERT(pop != NULL);
 
 	char opt = argv[1][0];
-	VALGRIND_PARTIAL_REORDER;
+	VALGRIND_EMIT_LOG(PARTIAL_ENGINE);
```

Comment:
I think i'd prefer to have a complete set of macros. So, instead of doing `VALGRIND_EMIT_LOG(PARTIAL_ENGINE`)`, you would do `VALGRIND_REORDER_PARTIAL`, `VALGRIND_REORDER_FULL`, etc.